src/Rings/mpolyquo-localizations.jl

```diff
@@ -2155,4 +2155,60 @@ function inverse(phi::MPolyQuoLocalizedRingHom)
   error("computation of inverse not implemented")
 end
 
+@doc raw"""
+    minimal_generating_set(I::MPolyLocalizedIdeal)
+
+Given an ideal `I` in the localization of a quotient of a  multivariate
+polynomial ring over a field at a point, return an array containing a
+minimal set of generators of `I`. If `I` is the zero ideal an empty list
+is returned.
+"""
+function minimal_generating_set(
+    I::MPolyQuoLocalizedIdeal{<:MPolyQuoLocRing{<:Field, <:FieldElem,
+                                          <:MPolyRing, <:MPolyElem,
+                                          <:MPolyComplementOfKPointIdeal},
+                              <:Any,<:Any}
+  )
+  Q = base_ring(I)
+  L = localized_ring(Q)
+
+  J = ideal(L, lift.(gens(I))) + modulus(Q)
```

Comment:
Shouldn't there also be a shift here? This method does not seem to do anything useful for now. Is this code still due to something I did?

Reply:
This is pulling the ideal in the localized quotient ring back to the localized ring, where we fall back to the method there. The shift is going on in the ring L not in Q -- to keep the ugly code at one specific place and not pour it over many files.

Reply:
I replaced it by pre_image_ideal. Thanks for the pointer.